src/runtime/virtcontainers/hypervisor.go

```diff
@@ -535,6 +538,16 @@ func (conf *HypervisorConfig) CustomImageAsset() bool {
 	return conf.isCustomAsset(types.ImageAsset)
 }
 
+// IgvmAssetPath returns the guest image path
+func (conf *HypervisorConfig) IgvmAssetPath() (string, error) {
+	return conf.assetPath(types.IgvmAsset)
+}
+
+// CustomIgvmAsset returns true if the image asset is a custom one, false otherwise.
```

Comment:
is this a function that is called somehow? I don't see a caller

Reply:
This is called in clh.go to get the igvm path